src/main/java/app/dbService/PostgresBookService.java

```diff
@@ -0,0 +1,120 @@
+package app.dbService;
+
+import app.dbService.dao.BookDao;
+import app.dbService.entity.Book;
+import org.h2.jdbcx.JdbcDataSource;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class PostgresBookService implements BookService {
+    private final static Connection connection = getPostgresConnection();
+    private BookDao dao;
+
+    public PostgresBookService() {
+        dao = new BookDao(connection);
+    }
+
+    private static Connection getPostgresConnection() {
+        try {
+            if (connection == null || connection.isClosed()) {
+                DriverManager.registerDriver((Driver) Class.forName("org.postgresql.Driver").newInstance());
+                String url = "jdbc:postgresql://localhost:5432/books";
+                String name = "postgres";
+                String pass = "5325475lol";
+
+                JdbcDataSource ds = new JdbcDataSource();
+                ds.setURL(url);
+                ds.setUser(name);
+                ds.setPassword(pass);
+
+                Connection connection = DriverManager.getConnection(url, name, pass);
+                return connection;
+            }
+        } catch (SQLException | InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+            Logger.getGlobal().info("Connection failed.");
+        }
+        return null;
+    }
+
+    protected void disconnect() throws SQLException {
+        if (connection != null && !connection.isClosed()) {
+            connection.close();
+        }
+    }
+
+    @Override
+    public void addBook(Book book) throws DBException {
+        try {
+            if (connection != null) {
+                connection.setAutoCommit(false);
```

Comment:
you dont need autocommit = false.  That kind of opsimisation  on current stage of developement is too much. When dealing with real world applications, the stages of optimisations are (in most cases):
1) Replace ORM with pure jdbc (already done)
2) Refactor current sql/nosql/cypher scripts
3) Search for such small code parts, that cause performance degradation
In most of the cases, step 1 already yields necessary 'gains' in performance.